docs/source/dashboard.rst

```diff
@@ -61,27 +80,53 @@ indicate the following.
 
     <table>
         <tr>
+            <th colspan="2">For individual worker bars</th>
+            <th colspan="2">For cluster bar</th>
+        </tr>
+        <tr>
+            <td>
+                <div role="img" aria-label="blue square" style="color:rgba(0, 0, 255, 1); font-size: 25px ">&#9632;</div>
+            </td>
+            <td>Memory under target threshold (In above example - 250MiB)</td>
             <td>
                 <div role="img" aria-label="blue square" style="color:rgba(0, 0, 255, 1); font-size: 25px ">&#9632;</div>
             </td>
-            <td>Memory under target (default 60% of memory available) </td>
+            <td>Memory under target threshold (In above example - 250MiB)</td>
         </tr>
         <tr>
             <td>
                 <div role="img" aria-label="orange square" style="color:rgba(255, 165, 0, 1); font-size: 25px ">&#9632;</div>
             </td>
-            <td> Memory is close to the spilling to disk target (default 70% of memory available)</td>
+            <td> Memory is close to the spilling to disk target (default above 50% of memory available)</td>
+            <td>
+                <div role="img" aria-label="orange square" style="color:rgba(255, 165, 0, 1); font-size: 25px ">&#9632;</div>
+            </td>
+            <td> Memory is close to the spilling to disk target by at least one worker - (default above 50% of memory available)</td>
```

Comment:
@crusaderky did the 70% targe default change? Can you verify how the default values relate to the color description here?

Reply:
It used to turn yellow at 50%; it's now at target: https://github.com/dask/distributed/pull/6959